src/lib/Paths.js

```diff
@@ -30,34 +31,47 @@ export class Paths {
     let pi = 0;
     let oi = 0;
     let search = true;
+    let closestIntersection = {}
 
     while(search) {
       
       let p = path.data[pi];
       let o = otherPath.data[oi];
-      let intersection = false;
+      let intersectionFound = false;
 
       // check time and location intersection
       if (fuzzyEqual(o.time, p.time, timePadding) &&
           fuzzyEqual(o.lat,  p.lat,  locationPadding) &&
           fuzzyEqual(o.long, p.long, locationPadding)) {
-            intersections.push({ 
+            let intersection = { 
               time: p.time,
               distance: distance(p.lat, p.long, o.lat, o.long),
-            })
-            intersection = true
+            }
+            intersections.push(intersection)
+
+            // keep track of closest intersection
+            if (closestIntersection.time === undefined || closestIntersection.time <= intersections.time ) {
```

Comment:
Since this is for the notification, I'm assuming it's closest by distance not by time, so please replace `time` with `distance` in this `if` statement

Reply:
Fixed